src/rpc/blockchain.cpp

```diff
@@ -2705,9 +2705,10 @@ static RPCHelpMan dumptxoutset()
 {
     return RPCHelpMan{
         "dumptxoutset",
-        "Write the serialized UTXO set to disk.",
+        "Write the UTXO set to disk.",
         {
             {"path", RPCArg::Type::STR, RPCArg::Optional::NO, "Path to the output file. If relative, will be prefixed by datadir."},
+            {"human_readable", RPCArg::Type::BOOL, RPCArg::Default{false}, "Dump file is created in human-readable format"}
```

Comment:
I think it would be better to have a non-Boolean param for this. Not only are Booleans terrible for positional arguments, but it's quite plausible in the future the format might change, and specifying a format version Number or format String would help ease the transition from the deprecated format.

Reply:
@luke-jr 

What do you think about shifting this boolean param from positional to option argument?

Reply:
I still don't think a boolean is a good fit either way.